### ISSUE TITLE
POVD.com not matching titles with concurrent special characters

### ISSUE THREAD

**Issue by covrdinbeez** [PAhelper/PhoenixAdult.bundle#824]
When searching to match this video, the agent seems to be stripping out the second dash. This results in it not finding the title.

Actual video URL: "https://povd.com/video/loft-fucking--2"
Search string: "POVD loft-fucking--2"
Log entries:
```
2020-06-25 13:56:21,637 (7000091f3000) :  INFO (agentkit:961) - Searching for matches for {'year': '2014', 'id': '715245', 'name': 'POVD loft-fucking--2'}
2020-06-25 13:56:21,637 (7000091f3000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/715245/tree'
2020-06-25 13:56:21,640 (7000091f3000) :  INFO (__init__:1073) - *******MEDIA TITLE****** POVD loft fucking 2
2020-06-25 13:56:21,640 (7000091f3000) :  INFO (__init__:1073) - Getting Search Settings for: POVD loft fucking 2
2020-06-25 13:56:21,640 (7000091f3000) :  INFO (PAsearchSites:1077) - mediaTitle w/ possible abbreviation: POVD loft fucking 2
2020-06-25 13:56:21,645 (7000091f3000) :  INFO (PAsearchSites:1336) - mediaTitle w/ possible abbrieviation fixed: POVD loft fucking 2
2020-06-25 13:56:21,647 (7000091f3000) :  INFO (PAsearchSites:1029) - Site found with method #3
2020-06-25 13:56:21,647 (7000091f3000) :  INFO (PAsearchSites:1336) - ^^^^^^^ siteID: 479
2020-06-25 13:56:21,647 (7000091f3000) :  INFO (PAsearchSites:1336) - ^^^^^^^ Shortening Title
2020-06-25 13:56:21,647 (7000091f3000) :  INFO (PAsearchSites:1336) - povd vs povd
2020-06-25 13:56:21,647 (7000091f3000) :  INFO (PAsearchSites:1336) - 1
2020-06-25 13:56:21,648 (7000091f3000) :  INFO (PAsearchSites:1348) - 8
2020-06-25 13:56:21,648 (7000091f3000) :  INFO (PAsearchSites:1360) - searchTitle (before date processing): loft fucking 2
2020-06-25 13:56:21,648 (7000091f3000) :  INFO (__init__:1073) - Search Title: loft fucking 2
2020-06-25 13:56:21,648 (7000091f3000) :  INFO (__init__:1073) - loft%20fucking%202
2020-06-25 13:56:21,650 (7000091f3000) :  DEBUG (networking:138) - Fetching 'https://povd.com/video/loft-fucking-2' from the HTTP cache
2020-06-25 13:56:21,654 (7000091f3000) :  CRITICAL (agentkit:1018) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'year': '2014', 'id': '715245', 'name': 'POVD loft-fucking--2'} (most recent call last):
```

**Comment by DirtyRacer1337** [comment]
Fixed in dd8cb5c00553a3b2622bca73e449a7c5e2cc60bd

**Comment by covrdinbeez** [comment]
Thanks. I loaded the new plugin and it now finds the movie and loads metadata for it. Only one problem. THe title it gives for the movie is "Loft Fucking [POVD] " (note there is a trailing space)
Looks like it loads all the other metadata properly.
Log from recent match:
2020-06-25 16:12:22,011 (70000ea69000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/715260/tree'
2020-06-25 16:12:22,014 (70000ea69000) :  INFO (__init__:1079) - *******MEDIA TITLE****** Loft Fucking
2020-06-25 16:12:22,015 (70000ea69000) :  INFO (__init__:1085) - Getting Search Settings for: Loft Fucking
2020-06-25 16:12:22,015 (70000ea69000) :  INFO (PAsearchSites:1077) - mediaTitle w/ possible abbreviation: Loft Fucking
2020-06-25 16:12:22,020 (70000ea69000) :  INFO (PAsearchSites:1289) - mediaTitle w/ possible abbrieviation fixed: Loft Fucking
2020-06-25 16:12:22,024 (70000ea69000) :  INFO (PAsearchSites:1360) - searchTitle (before date processing): Loft Fucking
2020-06-25 16:12:22,024 (70000ea69000) :  INFO (__init__:1085) - Search Title: Loft Fucking
2020-06-25 16:12:22,024 (70000ea69000) :  INFO (__init__:1085) - Loft%20Fucking
2020-06-25 16:12:22,026 (70000ea69000) :  DEBUG (runtime:88) - Sending packed state data (5336 bytes)
2020-06-25 16:12:22,026 (70000ea69000) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
2020-06-25 16:12:30,619 (70000ea69000) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcxNTI2MHMyCmlkczIwClBPVkQgbG9mdC1mdWNraW5nLS0yczQKbmFtZXM0CjIwMTRzNAp5ZWFycjAK
2020-06-25 16:12:30,619 (70000ea69000) :  DEBUG (runtime:49) - Received packed state data (5304 bytes)
2020-06-25 16:12:30,621 (70000ea69000) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcxNTI2MHMyCmlkczIwClBPVkQgbG9mdC1mdWNraW5nLS0yczQKbmFtZXM0CjIwMTRzNAp5ZWFycjAK
2020-06-25 16:12:30,621 (70000ea69000) :  INFO (agentkit:961) - Searching for matches for {'year': '2014', 'id': '715260', 'name': 'POVD loft-fucking--2'}
2020-06-25 16:12:30,621 (70000ea69000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/715260/tree'
2020-06-25 16:12:30,625 (70000ea69000) :  INFO (__init__:1079) - *******MEDIA TITLE****** POVD loft fucking 2
2020-06-25 16:12:30,625 (70000ea69000) :  INFO (__init__:1085) - Getting Search Settings for: POVD loft fucking 2
2020-06-25 16:12:30,625 (70000ea69000) :  INFO (PAsearchSites:1077) - mediaTitle w/ possible abbreviation: POVD loft fucking 2
2020-06-25 16:12:30,630 (70000ea69000) :  INFO (PAsearchSites:1289) - mediaTitle w/ possible abbrieviation fixed: POVD loft fucking 2
2020-06-25 16:12:30,632 (70000ea69000) :  INFO (PAsearchSites:1029) - Site found with method #3
2020-06-25 16:12:30,632 (70000ea69000) :  INFO (PAsearchSites:1305) - ^^^^^^^ siteID: 479
2020-06-25 16:12:30,632 (70000ea69000) :  INFO (PAsearchSites:1306) - ^^^^^^^ Shortening Title
2020-06-25 16:12:30,632 (70000ea69000) :  INFO (PAsearchSites:1307) - povd vs povd
2020-06-25 16:12:30,632 (70000ea69000) :  INFO (PAsearchSites:1311) - 1
2020-06-25 16:12:30,632 (70000ea69000) :  INFO (PAsearchSites:1348) - 8
2020-06-25 16:12:30,633 (70000ea69000) :  INFO (PAsearchSites:1360) - searchTitle (before date processing): loft fucking 2
2020-06-25 16:12:30,633 (70000ea69000) :  INFO (__init__:1085) - Search Title: loft fucking 2
2020-06-25 16:12:30,633 (70000ea69000) :  INFO (__init__:1085) - loft%20fucking%202
2020-06-25 16:12:30,634 (70000ea69000) :  DEBUG (networking:143) - Requesting 'https://povd.com/video/loft-fucking--2'
2020-06-25 16:12:31,441 (70000ea69000) :  DEBUG (runtime:88) - Sending packed state data (5336 bytes)
2020-06-25 16:12:31,441 (70000ea69000) :  DEBUG (runtime:924) - Response: [200] str, 660 bytes
2020-06-25 16:12:32,223 (70000ea69000) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgo3MTUyNjBzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzNjEKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL2xvZnQtZnVja2luZy0yfDQ3OXw%40bGFuZz1lbnM0Cmd1aWRzMTkKbG9mdC1mdWNraW5nLTJ8NDc5fHMyCmlkcjAK
2020-06-25 16:12:32,224 (70000ea69000) :  DEBUG (runtime:49) - Received packed state data (22224 bytes)
2020-06-25 16:12:32,227 (70000ea69000) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNgo3MTUyNjBzNApkYmlkaTAKczcKdmVyc2lvbm5zMTAKcGFyZW50R1VJRG5zOApwYXJlbnRJRHM1Ck1vdmllczEwCm1lZGlhX3R5cGVzNjEKY29tLnBsZXhhcHAuYWdlbnRzLnBob2VuaXhhZHVsdDovL2xvZnQtZnVja2luZy0yfDQ3OXw@bGFuZz1lbnM0Cmd1aWRzMTkKbG9mdC1mdWNraW5nLTJ8NDc5fHMyCmlkcjAK
2020-06-25 16:12:32,227 (70000ea69000) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://loft-fucking-2|479|?lang=en
2020-06-25 16:12:32,228 (70000ea69000) :  DEBUG (model:234) - Deserializing from /Users/*******/Library/Application Support/Plex Media Server/Metadata/Movies/6/ed2d1557feecbf5339ece9fb12d9709e2f8ad55.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2020-06-25 16:12:32,229 (70000ea69000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/715260/tree'
2020-06-25 16:12:32,233 (70000ea69000) :  INFO (__init__:1103) - ******UPDATE CALLED*******
2020-06-25 16:12:32,233 (70000ea69000) :  INFO (__init__:1106) - 479
2020-06-25 16:12:32,233 (70000ea69000) :  INFO (networkPornPros:32) - scene url: https://povd.com/video/loft-fucking-2
2020-06-25 16:12:32,234 (70000ea69000) :  DEBUG (networking:143) - Requesting 'https://povd.com/video/loft-fucking-2'
2020-06-25 16:12:36,721 (70000ea69000) :  CRITICAL (agentkit:1095) - Exception in the update function of agent named 'PhoenixAdult', called with guid 'com.plexapp.agents.phoenixadult://loft-fucking-2|479|?lang=en' (most recent call last):
  File "/Applications/Plex Media Server.app/Contents/Resources/Plug-ins-79e214ead/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1093, in _update
    agent.update(obj, media, lang, **kwargs)
  File "/Users/*******/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 2162, in update
    metadata = PAsearchSites.siteMylf.update(metadata, siteID, movieGenres, movieActors)
  File "/Users/*******/Library/Application Support/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/networkPornPros.py", line 167, in update
    fanSite = PAextras.getFanArt("Tiny4KFan.com", art, actors, actorName, metadata.title, 0, siteName)
  File "/Applications/Plex Media Server.app/Contents/Resources/Plug-ins-79e214ead/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 108, in <lambda>
    _getitem_           = lambda x, y: x.__getitem__(y),
IndexError: list index out of range

2020-06-25 16:12:36,722 (70000ea69000) :  DEBUG (model:229) - Serializing to /Users/*******/Library/Application Support/Plex Media Server/Metadata/Movies/6/ed2d1557feecbf5339ece9fb12d9709e2f8ad55.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2020-06-25 16:12:36,726 (70000ea69000) :  DEBUG (runtime:88) - Sending packed state data (22256 bytes)
2020-06-25 16:12:36,726 (70000ea69000) :  DEBUG (runtime:924) - Response: [200] str, 16 bytes

**Comment by DirtyRacer1337** [comment]
Fixed in a7fa652db0f8cee4c43d269c3a10e00550abac21
